packages/lib-classifier/src/components/Classifier/components/Layout/Layout.js

```diff
@@ -1,20 +1,27 @@
 import { observer } from 'mobx-react'
-
 import { useStores } from '@hooks'
 import getLayout from './helpers/getLayout'
 
 function storeMapper(classifierStore) {
   const workflow = classifierStore.workflows.active
+  const subject = classifierStore.subjects.active
 
   return {
-    layout: workflow?.layout
+    multiFrameSubject: subject?.isMultiFrameSubject,
```

Comment:
The big change in this PR is that layout is no longer fixed when the workflow loads. Layout now observes the active subject and re-renders. 

This means that layout can change from one subject to the next as we work through a workflow eg. a camera trap subject set with both multiple frames and single frame subjects will switch layout from one subject to the next.

Reply:
**Is a non-static layout a blocker for this PR?**

If a non-static layout IS a blocker for this PR, I think our only other option is to modify MaxWidth layout component to directly handle when `flipbookMode` is set to `separate` in the subject viewer store. That means there will be a lot of code in the MaxWidth layout that does not apply to non-flipbook projects.

If a non-static layout is NOT a blocker for this PR, I don't know of any live projects on FEM that meet the following criteria: 1. Non-transcription project and 2. Some subjects have 1 location and others have >1 location. Do you have an example project that meets those criteria that is launched on FEM or about to be launched?

Reply:
Another thought - In the case where a non-static layout IS a blocker, we have the option of taking ImageToolbar and MetaTools out of the layout components, and placing them in each subject viewer's container instead. The subject viewer would control when to display 1 or more than 1 image toolbar. 

That would be an even larger change than this current PR, but could be worth it for FEM's longevity. The only thing I think would be super difficult in that scenario is changing where the field guide is displayed based on default versus separate frames view.

Reply:
>If a non-static layout is NOT a blocker for this PR, I don't know of any live projects on FEM that meet the following criteria: 1. Non-transcription project and 2. Some subjects have 1 location and others have >1 location. Do you have an example project that meets those criteria that is launched on FEM or about to be launched?

Nothing on FEM but Snapshot Serengeti or similar projects have a mixture of subjects like this.

Reply:
> Is a non-static layout a blocker for this PR?

^ Thoughts on this question? We could also get around the problem of refreshing the entire layout by creating a ViewerGrid component for non-multi-frame images and a ViewerGrid component specifically for multi-frame images.

Reply:
I’m not sure, after a quick look at the changes (using PH-TESS.) Should it be a blocking issue?

Potential blockers, to me, are:
- unintended side effects on workflows that don’t need the new layout, like PH-TESS. Those can be avoided by more restrictive reflection on the type of subject. I think we can also check the workflow to see if the project team have specifically enabled the new layout in the project builder. 
- any inconsistent classification UX  from switching between layouts while classifying eg. the Field Guide button should be in the same place for every subject. I didn’t see any layout changes for PH-TESS but I’m not sure what to look for.

Reply:
Noting here that on camera trap subjects like Snapshot Serengeti, with a mix of subject types, we currently show the flipbook/separate frame toggle even  when there's only one frame from the camera trap. So the flipbook/separate frame view is something that applies to the workflow as a whole, rather than individual subjects. [Snapshot Venetia](https://www.zooniverse.org/projects/shuebner729/snapshot-venetia/classify) is an example.